docs/howto/admin/resource-estimation.rst

```diff
@@ -58,11 +59,13 @@ stop, unlike with RAM.
 
 .. math::
 
-    Server CPU Recommended = (Maximum concurrent users \times Maximum CPU usage per user) + 0.2
+    Recommended\, CPU = (Max\, concurrent\, users \times Max\, CPU\, usage\, per\, user) + 20\%
 
-The ``0.2`` is overhead for TLJH and related services. **Server CPU Recommended**
-is the amount of CPU the server you acquire should have. We recommend using
+The ``20%`` is overhead for TLJH and related services. This is around 20% of a 
+single modern CPU. This of course is just an estimate. We recommend using
```

Comment:
I think that a couple of commas are missing: 

`This, of course, is just an estimate`